game/scripts/npc/items/item_illusionsts_cape.txt

```diff
@@ -0,0 +1,96 @@
+    "BaseClass"                                           "item_datadriven"
+    "Model"                                               "models/props_gameplay/recipe.mdl"
+    "FightRecapLevel"                                     "1"
+    "ItemCost"                                            "20000"
+    "ItemShopTags"                                        ""
+    "ItemRecipe"                                          "1"
+    "ItemResult"                                          "item_illusionsts_cape"
+    "AbilityTextureName"                                  "custom/recipe/recipe_5"
+    "ItemRequirements"
+    {
+      "01"                                                "item_manta_4;item_upgrade_core_4"
+    }
+  }
+
+  //=================================================================================================================
+  // Illusionist Cape
+  //=================================================================================================================
+  "item_illusionsts_cape"
+  {
+    // General
+    //-------------------------------------------------------------------------------------------------------------
+    "ID"                                                  "363" // unique ID number for this item.  Do not change this once established or it will invalidate collected stats.
+    "AbilityBehavior"                                     "DOTA_ABILITY_BEHAVIOR_IMMEDIATE | DOTA_ABILITY_BEHAVIOR_NO_TARGET"
+
+    // Stats
+    //-------------------------------------------------------------------------------------------------------------
+    "AbilityCooldown"                                     "30.0"
+
+    // Item Info
+    //-------------------------------------------------------------------------------------------------------------
+    "ItemCost"                                            "37700" //OAA
+    "ItemShopTags"                                        "agi;str;hard_to_tag"
+    "ItemIsNeutralDrop"                                   "0" //OAA
+    "ItemPurchasable"                                     "1" //OAA
+    "ItemDisassembleRule"                                 "DOTA_ITEM_DISASSEMBLE_NEVER"
+    "ItemAliases"                                         "illusion cape;illusionist;illusionist cape"
+    "ItemDeclarations"                                    "DECLARE_PURCHASES_TO_TEAMMATES | DECLARE_PURCHASES_IN_SPEECH | DECLARE_PURCHASES_TO_SPECTATORS"
+    "ItemShareability"                                    "ITEM_NOT_SHAREABLE"
+
+    "Model"                                               "models/props_gameplay/neutral_box.vmdl"
+
+    // Special
+    //-------------------------------------------------------------------------------------------------------------
+    "AbilitySpecial"
+    {
+      "01"
+      {
+        "var_type"                                        "FIELD_INTEGER"
+        "attack_damage_aura"                              "10"
```

Comment:
Illu. Cape and Manta dont share cd. I thought I made that clear. This attack damage aura also works on player controlled creeps. Ilu. Cape having bad numbers is your fault because you buffed Manta illusion numbers.

Reply:
ima be honest. This is just a better version of manta (nobody really buys manta for the debuff purge). Honestly if there was some way to combine the effects of Illusionist cape and put it on manta. (Alongside the illusions, you have an aura that increases illusion dmg and stuff) that would honestly make manta a good item again.